apis/delivery/parsers.py

```diff
@@ -0,0 +1,4 @@
+from flask_restplus import reqparse
+
+delivery_parser = reqparse.RequestParser(bundle_errors=True)
+delivery_parser.add_argument('listing_hash', type=str, required=True, location='args', help='The listing hash requested for this delivery')
```

Comment:
should be `delivery_hash` not `listing_hash` right?

Reply:
delivery_hash added in addition to listing_hash